GameServerLib/GameObjects/AttackableUnits/AI/ObjAIBase.cs

```diff
@@ -479,7 +479,7 @@ public override void OnCollision(IGameObject collider)
 
         public bool RecalculateAttackPosition()
         {
-            if (Target != null && TargetUnit != null && !TargetUnit.IsDead && GetDistanceTo(Target) < CollisionRadius && GetDistanceTo(TargetUnit.X, TargetUnit.Y) <= Stats.Range.Total) //If we are already where we should be, do not move.
+            if (Target != null && TargetUnit != null && !TargetUnit.IsDead && GetDistanceTo(Target) > CollisionRadius && GetDistanceTo(TargetUnit.X, TargetUnit.Y) <= Stats.Range.Total) //If we are already where we should be, do not move.
```

Comment:
Can you describe why you changed the sign?
I’m not familiar with this code.

Reply:
The previous sign would mean that we don't move if we are intersecting with something and we are in auto attack range. The new sign would mean that we don't move if we aren't intersecting with something and we are in auto attack range. This helps reduce collision calls.

Reply:
Does this mean the minion will attempt to move if it's standing on top of something else?

Reply:
Correct, in order to stop colliding.

Reply:
That's very cool. I'll keep my eye out to watch for that when I'm testing.

Reply:
Could you add this as a code comment about the line you changed? It would kick travis into re-building since it got stuck.

You can use github's direct file edit ability to add the change to the PR if you want to do it quickly. No need for an IDE to add a comment.